src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -65,7 +65,19 @@ def make_artifacts(
     is_flag=True,
     help="Drop into python debugger if an error occurs.",
 )
-def make_results(output_dir: str, verbose: int, with_debugger: bool) -> None:
+@click.option(
+    "-b",
+    "--mark-best",
+    is_flag=True,
```

Comment:
I think `is_flag`s default to True. Probably want to add default=False

Reply:
Nope it defaults to False